DICOM/src/Microsoft.Health.DeID.SharedLib/EncryptFunction.cs

```diff
@@ -7,16 +7,22 @@
 using System.IO;
 using System.Security.Cryptography;
 using System.Text;
-using Newtonsoft.Json;
+using Microsoft.Health.DeID.SharedLib.Settings;
 
 namespace Microsoft.Health.Dicom.DeID.SharedLib
 {
     public class EncryptFunction
     {
         // AES Initialization Vector length is 16 bytes
         private const int AesIvSize = 16;
+        private EncryptionSetting _encryptionSetting;
 
-        public static byte[] EncryptContentWithAES(string plainText, byte[] key, Encoding encoding = null)
+        public EncryptFunction(EncryptionSetting encryptionSetting)
```

Comment:
can not change key for EncryptFunction?

Reply:
one EncryptionFunction own one key as property.